UI/ChatGPT/src/ChatGPT/Services/ChatService.cs

```diff
@@ -43,56 +64,47 @@ public async IAsyncEnumerable<ChatResponse> AskAsStream(ChatRequest chatRequest,
 		var response = new ChatResponse();
 		var content = new StringBuilder();
 
-		IAsyncEnumerator<ChatCompletionCreateResponse>? responseStream = default;
-		while (!response.IsError)
+		AsyncResultCollection<StreamingChatCompletionUpdate> result = _client.CompleteChatStreamingAsync(request);
+		var enumerator = result.GetAsyncEnumerator(ct);
+
+		try
```

Comment:
The `_client.CompleteChatStreamingAsync(request);` is no longer try/catch, so network exception will no longer be reported in the chat (they will instead cause the exception to bubble up to the `AskMessage`  command in the `MainModel` (and as there is no handler at this point they will only be logged). Please restore back the `responseStream ??= _client.CreateCompletionAsStream(request).GetAsyncEnumerator(ct);`  into the `try/catch` 

cc. @Xiaoy312